settings.py

```diff
@@ -892,6 +893,8 @@
 SENTRY_QUERY_URL = 'https://sentry.io/{org}/{project}/?query='
 SENTRY_API_KEY = None
 
+ENABLE_PASSWORD_HASHING = False
```

Comment:
I know this is annoying and touches a lot of different places, but could we change the name of this setting and related functions to

1. include the phrase `NIC_COMPLIANCE` so it's clear that this is tied to an NIC compliance feature and
2. use a word like 'obfuscation' instead of 'hashing'?

Unless calling it 'hashing' is itself a requirement for NIC compliance, I'd prefer to call it obfuscating because hashing is the concept of taking a very large/infinite input space (such as "all strings") and mapping it to very (comparatively) small output space (such as "all 128-bit integers"); moreover colloquially it implies the properties we associate with a "good" hashing function (such as consistent distribution), and in the context of passwords specifically, with those of a cryptographic hashing function (such as bit scrambling and irreversibility). Since the obfuscation scheme we're using for NIC compliance doesn't have any of those properties, I think simply calling it "obfuscation" would lead to less confusion/surprise.